_raml/apps/products/schema/post_product_request.schema.json

```diff
@@ -4,7 +4,7 @@
   "type": "object",
   "properties": {
     "": {
-      "type": "object of [product (create or update request)](page:apps-data-types#product-create-or-update-request)",
+      "type": "object of [product (create or update request)](page:apps-data-types#product-create-request)",
```

Comment:
remove *or update* there as well:
object of [product (create or update request)]